app/controllers/api_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ApiController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  private
+  def authenticated?
+    authenticate_or_request_with_http_basic { |name, password| User.where( name: name, password: password).present? }
```

Comment:
small nitpicky thing, watching spacing on this one... User.where(name: name, password: password).present?